terasoluna-gfw-functionaltest-domain/src/main/java/org/terasoluna/gfw/functionaltest/domain/service/logging/TraceLoggingInterceptorServiceImpl.java

```diff
@@ -15,18 +15,24 @@
  */
 package org.terasoluna.gfw.functionaltest.domain.service.logging;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
 @Service
 public class TraceLoggingInterceptorServiceImpl implements
                                                 TraceLoggingInterceptorService {
 
+    private static final Logger logger = LoggerFactory.getLogger(
+            TraceLoggingInterceptorServiceImpl.class);
+
     @Override
     public void sleep(long millis) {
         try {
             Thread.sleep(millis);
         } catch (InterruptedException e) {
-            e.printStackTrace();
+            logger.warn(e.getMessage());
```

Comment:
- [ ] You may select log level `debug` or `trace`.
Because the developer intensionaly sleep thread and `InterruptedException` has no meaning here.

Reply:
Change log level to `trace`.
8d8585655f02a10249b9b28a96f7b05d435e13db